modules/devguide/examples/c/bulk-get.cc

```diff
@@ -92,6 +93,7 @@ main()
     check(lcb_wait(instance, LCB_WAIT_DEFAULT), "wait for batch to complete");
 
     for (auto& result : results) {
+    // end::batching
```

Comment:
Just checking: is it intentional that the tag ends in the middle of a for loop?

Reply:
It's missing the terminating [] in any case, and thus not appearing in the .adoc

Reply:
No probably, it should be moved. I will check